website/content/en/preview/reference/cloudformation.md

```diff
@@ -390,6 +396,7 @@ Also, `karpenter.k8s.aws/ec2nodeclass` must be set to some value. This ensures t
   "Condition": {
     "StringEquals": {
       "aws:RequestTag/kubernetes.io/cluster/${ClusterName}": "owned",
+      "aws:RequestTag/eks:eks-cluster-name": "${ClusterName}",
```

Comment:
Do we need to make an update to the section above to include the fact that we are now requiring that the CloudFormation set the `eks:eks-cluster-name` tag in the request?

Reply:
Done.